rfc6962-bis.xml

```diff
@@ -930,7 +936,7 @@ an accepted root certificate.
             </list></t>
           </list>
 
-          Both tree sizes must be from existing v1 STHs (Signed Tree Heads).
+          Both tree sizes must be from existing v1 STHs (Signed Tree Heads). However, because of skew, the receiving server may not know one or both of the existing STHs. If both are known, then only the <spanx style="verb">consistency</spanx> output is returned. If the first is known but not the second, then the latest known STH is returned, along with a consistency proof between the first STH and the latest.
```

Comment:
Presumably if both STHs are unknown consistency will be entirely omitted and only the latest known STH is returned?
Also, nit: How about using the term 'front-end' rather than 'server' to disambiguate the 'CT log server' entity from its components?